licenses.yaml

```diff
@@ -364,12 +364,32 @@ name: Guava
 license_category: binary
 module: java-core
 license_name: Apache License version 2.0
-version: 16.0.1
+version: 31.1-jre
 libraries:
   - com.google.guava: guava
 
 ---
 
+name: Failureaccess
+license_category: binary
+module: java-core
+license_name: Apache License version 2.0
+version: 1.0.1
+libraries:
+  - com.google.guava: failureaccess
+
+---
+
+name: listenablefuture
+license_category: binary
+module: java-core
+license_name: Apache License version 2.0
+version: 9999.0-empty-to-avoid-conflict-with-guava
```

Comment:
why is this change required?

Reply:
Apparently this is a transitive dependency of `com.google.guava:guava:31.1-jre`. Although this is an empty artifactory I am not entirely sure of repurcusions of removing this dependency.